application/backend/src/main/resources/apps/typerefinery/components/content/barchart/template.html

```diff
@@ -0,0 +1,11 @@
+<template data-sly-template.barchart="${ @ model }">
```

Comment:
camel case convention should be used here.

barchart to barChart

Reply:
done